### PR TITLE
DOCS-1217 RUM Browser Monitoring

### PR DESCRIPTION
**Pull request by ruthnaebeck**
## Motivation

Single source documentation

## Changes

- Combine information from current README.md and [docs site](https://docs.datadoghq.com/real_user_monitoring/browser)
- Update CODEOWNERS

## Testing

https://docs-staging.datadoghq.com/ruth/docs-1217/real_user_monitoring/browser/

### CODE REVIEW

**Comment by BenoitZugmeyer** [comment]
Great idea to sync our documentation! I'll try to make a small script to partially automate the process.

**Review by sarina-dd — APPROVED** [review]

**Comment by hdelaby** [comment]
Hey @ruthnaebeck thanks for this :)

Following [my comment on the other PR](https://github.com/DataDog/documentation/pull/8521#issuecomment-693525853),  we're missing EU/US differences in set up instructions.

Also I need your opinion :) Given that we already explore global context/add user actions APIs in the "advanced set up" section of the documentation (and soon in dedicated sections), my idea was to keep this page with the minimum amount of information to get it working. Is it a good idea to remove these sections?

**Comment by ruthnaebeck** [comment]
@hdelaby - I posted this comment in the docs repo:
> We could add a shortcode that will automatically change it based on the site selector but this won't display in the source repo. I opted to replace it with `<DATADOG_SITE>` because I expect people will copy/paste directly from the APP to get their unique ID codes.

Afterwards, I noticed the latest content wasn't in the preview link. That is being updated now.

Concerning this comment:
> Given that we already explore global context/add user actions APIs in the "advanced set up" section of the documentation (and soon in dedicated sections), my idea was to keep this page with the minimum amount of information to get it working. Is it a good idea to remove mentions of these APIs?

Makes sense to remove if it's already presented elsewhere.

**Comment by bcaudan** [comment]
@ruthnaebeck we should probably do the same for browser-logs ([doc](https://docs.datadoghq.com/logs/log_collection/javascript/?tab=npm), [github](https://github.com/DataDog/browser-sdk/blob/master/packages/logs/README.md)), should it be handled in a separate PR?

**Comment by hdelaby** [comment]
@ruthnaebeck RE: US/EU site => I trust you with the best approach :) 
RE: global context/user actions => let's remove it then

**Comment by ruthnaebeck** [comment]
> @ruthnaebeck we should probably do the same for browser-logs ([doc](https://docs.datadoghq.com/logs/log_collection/javascript/?tab=npm), [github](https://github.com/DataDog/browser-sdk/blob/master/packages/logs/README.md)), should it be handled in a separate PR?

@bcaudan - Yep, we can handle in a separate PR. I will add a card.

File: packages/rum/README.md [hunk 1]
```diff
@@ -107,19 +149,10 @@ parents) that will be used to name the action. Examples:
...
-  sampleRate: 100
-});
-```
+[1]: https://app.datadoghq.com/rum/list
+[2]: /real_user_monitoring/data_collected/
+[3]: /real_user_monitoring/dashboards/
+[4]: https://www.npmjs.com/package/@datadog/browser-rum
+[5]: /account_management/api-app-keys/#client-tokens
+[6]: /real_user_monitoring/data_collected/user_action/#automatic-collection-of-user-actions
+[7]: /real_user_monitoring/faq/proxy_rum_data/
```
Comment (bcaudan) [inline packages/rum/README.md:158]: relative links won't work from github, could we use full url instead?
ex: `/real_user_monitoring/data_collected/` -> `https://docs.datadoghq.com/real_user_monitoring/data_collected/`

**Review by bcaudan — APPROVED** [review]

**Comment by bcaudan** [comment]
@ruthnaebeck could you sign the CLA?
we could then merge this PR

**Pull request merged**